packages/yoroi-extension/app/api/thunk.js

```diff
@@ -404,9 +412,21 @@ const callbacks = Object.freeze({
   serverStatusUpdate: [],
   coinPriceUpdate: [],
 });
-chrome.runtime.onMessage.addListener(async (message, _sender, _sendResponse) => {
+chrome.runtime.onMessage.addListener((serializedMessage, _sender, _sendResponse) => {
   //fixme: verify sender.id/origin
-  Logger.debug('get message from background:', JSON.stringify(message, null, 2));
+  Logger.debug('get message from background:', serializedMessage);
+
+  let message;
+  try {
+    message = JSON.parse(serializedMessage);
+  } catch {
+    Logger.error('underializable message');
```

Comment:
```suggestion
  } catch (e) {
    Logger.error('unserializable message: ' + serializedMessage, e);
```

Reply:
The message has been logged on L417 and since the only thing in the `try` block is `JSON.parse`, I think the error can be easily inferred.

Reply:
The message logged on the L417 is in DEBUG, we need error logs as extended as possible for the case when users will be reporting issues. Very often we receive issue reports as only error messages from consoles or even their screenshots.

Not having full information about what exactly has happened in the error is completely developing-process-centric, thinking we can get into full debug mode at any moment, instead of thinking of cases when we gonna have minimal information possible from a user report with no clear way to reproduce what's failing.

Please always apply the simple rule of thumb:
1. DEBUG logs in the middle of **normal** flow are for tracing during development/investigation - they may contain minimal information needed
2. ERROR logs are only used in branches that are NOT supposed to happen at all in a normal case , so when this branch happens - AS MUCH information as possible must be clearly outputted to help any future investigation

Understanding this difference may literally save us days of investigation in the future.